app/javascript/chat/ActiveChatChannel/ChatMessages.js

```diff
@@ -3,7 +3,36 @@ import PropTypes from 'prop-types';
 import ActionMessage from '../actionMessage';
 import Message from '../message';
 import DirectChatInfoMessage from './IntroductionMessages/DireactIntroMessages';
-import OpenChatInfoMessagge from './IntroductionMessages/OpenItnroMessage';
+import IntroductionMessage from './IntroductionMessages/OpenItnroMessage';
```

Comment:
spelling (nitpick): OpenItnroMessage should be OpenIntroMessage

```suggestion
import IntroductionMessage from './IntroductionMessages/OpenIntroMessage';
```